public/app/features/alerting/unified/components/silences/SilencesEditor.tsx

```diff
@@ -97,24 +95,24 @@ const getDefaultFormValues = (searchParams: URLSearchParams, silence?: Silence):
   }
 };
 
-export const SilencesEditor = ({ silence, alertManagerSourceName }: Props) => {
+export const SilencesEditor = ({ silenceId, alertManagerSourceName }: Props) => {
+  const history = useHistory();
```

Comment:
(Maybe todo/note to self: sort out useHistory vs locationService? What's going on here?)

Reply:
I've refactored to use that now - I had to make some changes to the test render method that I recently added (but I expected this might have to happen as we start adopting the method and using it for different cases)